### PR TITLE
Pluto QAQC: Mismatch Graph Descriptions

### PR DESCRIPTION
**Pull request by abrieff**
Updates Description for Mismatch Graphs based on Amanda's Comments on #106

### CODE REVIEW

File: src/pluto/pluto.py [hunk 1]
```diff
@@ -189,27 +189,27 @@ def create_mismatch(df_mismatch, v1, v2, v3, condo, mapped):
 
             groups = [
                 {
-                    "title": "Mismatch graph -- finance",
+                    "title": "Mismatch graph -- finance fields",
```
Comment (SashaWeinstein) [inline src/pluto/pluto.py:192]: Can these titles be larger? I changed the `update_layout` call in in the for loop to 
```python
                fig.update_layout(
                    title={"text": group["title"], "font": dict(size=24)},
                    template="plotly_white",
                )
```
and thought it made the app easier to navigate

File: src/pluto/pluto.py [hunk 2]
```diff
@@ -281,7 +281,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
                 fig.update_layout(title=group["title"], template="plotly_white")
                 st.plotly_chart(fig)
                 st.info(group["description"])
+
             st.write(df)
+            st.info(
```
Comment (SashaWeinstein) [inline src/pluto/pluto.py:286]: Don't think this info is clearly associated with the table. The table could use it's own header that clearly breaks it off into it's own section

**Review by SashaWeinstein — COMMENTED** [review]
Close, left some comments

File: src/pluto/pluto.py (hunk 1 above)
Comment (SashaWeinstein) [reply src/pluto/pluto.py:192]: Alternatively if you don't feel like iterating on this we could skip it, it's a minor aesthetic thing and it seems like we have more interesting work to in the second week of this sprint

Comment (abrieff) [reply src/pluto/pluto.py:192]: Agreed that we should do that in the future (although I'm honestly tempted by having no title, and using the subheader tag instead) but think its better left for a different issue. Happy to write it up - increased the size on these for now

File: src/pluto/pluto.py (hunk 2 above)
Comment (abrieff) [reply src/pluto/pluto.py:290]: 👍

File: src/pluto/pluto.py (hunk 1 above)
Comment (abrieff) [reply src/pluto/pluto.py:192]: to your point though, i added a global color scheme constant as part of #113, could be same implementation

File: src/pluto/pluto.py [hunk 3]
```diff
@@ -278,10 +278,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
...
                 )
-                fig.update_layout(title=group["title"], template="plotly_white")
+                fig.update_layout(
+                    title={"text": group["title"], "font": dict(size=24)},
+                    template="plotly_white",
+                )
                 st.plotly_chart(fig)
                 st.info(group["description"])
 
+            st.subheader("Records with Differences by Field")
```
Comment (SashaWeinstein) [inline src/pluto/pluto.py:288]: I don't think this is the best header here as the table doesn't list records. I think we should replace "Records with" with "Summary of"

File: src/pluto/pluto.py [hunk 4]
```diff
@@ -278,10 +278,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
                 fig = generate_graph(
                     v1v2, v2v3, v1v2_total, v2v3_total, group["columns"]
                 )
-                fig.update_layout(title=group["title"], template="plotly_white")
+                fig.update_layout(
```
Comment (SashaWeinstein) [inline src/pluto/pluto.py:281]: I know I suggested this code, but the issue with these change is that the titles for the mismatch graphs are a different than the titles for aggregate, nulls, etc. Making the titles consistent would require a more complex implementation which I why I suggested we skip. Happy to help implement a DRY way to make the titles bigger if it's interesting to you Andrew

**Review by SashaWeinstein — COMMENTED** [review]
Close but still some things to clean up

File: src/pluto/pluto.py (hunk 4 above)
Comment (abrieff) [reply src/pluto/pluto.py:281]: fair enough, let's just save it for seperate issue to do them all. We probably want to combine it with in general making universal styling defaults - the plotly_white thing probably doesn't need to be everywhere either

Comment (abrieff) [reply src/pluto/pluto.py]: added #127

**Review by SashaWeinstein — APPROVED** [review]
Looks good to me!

**Pull request merged**